client/app/vps/dashboard/rebuild/vps-rebuild.service.js

```diff
@@ -0,0 +1,27 @@
+export default class VpsRebuild {
+  /* @ngInject */
+  constructor($q, OvhApiVps) {
+    this.$q = $q;
+    this.OvhApiVps = OvhApiVps;
+  }
+
+  getAvailableImages(serviceName) {
+    return this.OvhApiVps.Images().Available().v6()
```

Comment:
This call is not done in the state `resolve`, because of the long request to get all available images details